src/lib/swapper/swappers/LifiSwapper/endpoints.ts

```diff
@@ -103,7 +103,18 @@ export const lifiApi: Swapper2Api = {
     //   // Note, this may change if the Li.Fi SDK changes
     //   url: 'https://li.quest/v1/status',
     // })
-    const statusResponse = await getLifi().getStatus(getStatusRequest)
+
+    // don't use lifi sdk here because all status responses are cached, negating the usefulness of polling
+    // i.e don't do `await getLifi().getStatus(getStatusRequest)`
+    const url = new URL('https://li.quest/v1/status')
+    url.search = new URLSearchParams(
+      getStatusRequest as unknown as Record<string, string>,
+    ).toString()
```

Comment:
To remove this cast and make the types correct we might want to do the following when defining `getStatusRequest`:

```ts
const getStatusRequest: GetStatusRequest & Record<string, string> = {
  txHash,
  bridge: lifiRoute.steps[0].tool,
  fromChain: lifiRoute.fromChainId.toString(),
  toChain: lifiRoute.toChainId.toString(),
}
```

then we can just do

```suggestion
    url.search = new URLSearchParams(getStatusRequest).toString()
```